clients/ctl/admin-ui/src/features/dataset/ClassifiedDataCategoryDropdown.tsx

```diff
@@ -0,0 +1,138 @@
+import { Box, ButtonProps } from "@fidesui/react";
+import { MultiValue, Select } from "chakra-react-select";
+import { useMemo } from "react";
+
+import { DataCategory } from "~/types/api";
+
+import DataCategoryDropdown from "./DataCategoryDropdown";
+import type { Props as DataCategoryDropdownProps } from "./DataCategoryInput";
+
+// TODO: just making up a structure until we have something real from the API
+interface DataCategoryWithConfidence extends DataCategory {
+  confidence: number;
+}
+
+interface Props extends Omit<DataCategoryDropdownProps, "tooltip"> {
+  mostLikelyCategories: DataCategoryWithConfidence[];
+}
+
+const ClassifiedDataCategoryDropdown = ({
+  mostLikelyCategories,
+  onChecked,
+  checked,
+  dataCategories,
+}: Props) => {
+  const menuButtonProps: ButtonProps = {
+    size: "sm",
+    colorScheme: "complimentary",
+    borderRadius: "6px 0px 0px 6px",
+  };
+
+  const allCategoriesWithConfidence = useMemo(
+    () =>
+      dataCategories.map((dc) => {
+        const categoryWithConfidence = mostLikelyCategories.find(
+          (mlc) => mlc.fides_key === dc.fides_key
+        );
+        if (categoryWithConfidence) {
+          return categoryWithConfidence;
+        }
+        return { ...dc, confidence: undefined };
+      }),
+    [mostLikelyCategories, dataCategories]
+  );
+
+  const options = allCategoriesWithConfidence
+    .sort((a, b) => {
+      if (a.confidence !== undefined && b.confidence !== undefined) {
+        return b.confidence - a.confidence;
+      }
+      if (a.confidence === undefined) {
+        return 1;
+      }
+      if (b.confidence === undefined) {
+        return -1;
+      }
+      return a.fides_key.localeCompare(b.fides_key);
+    })
+    .map((c) => {
+      const confidence =
+        c.confidence === undefined ? "N/A" : `${c.confidence}%`;
```

Comment:
Could throw a `confidence.toFixed()` on there just in case the number is overly precise.